app/services/bill-runs/two-part-tariff/review-bill-run.service.js

```diff
@@ -14,11 +14,12 @@ const ReviewBillRunPresenter = require('../../../presenters/bill-runs/two-part-t
  * @param {String} id The UUID for the bill run to review
  * @param {Object} payload The `request.payload` containing the filter data. This is only passed to the service when
  * there is a POST request, which only occurs when a filter is applied to the results.
- *
+  * @param {Object} sessionManager - The Hapi `request.yar` session manager passed on by the controller
+*
```

Comment:
NIT

```suggestion
 * @param {Object} yar - The Hapi `request.yar` session manager passed on by the controller
 *
```

Reply:
Yes I didn't like that either :grin:. I copied it from the original one you did in `app/services/bill-runs/two-part-tariff/review-licence.service.js` so figured it was your convention for `yar`

Reply:
I've also sorted that out in `app/services/bill-runs/two-part-tariff/review-licence.service.js`